watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py

```diff
@@ -1310,26 +1309,30 @@ def model6():
 @pytest.mark.unit
 def test_elec_properties_errormsg(model6):
```

Comment:
Why does this test need to be changed/removed? I think there should be no interference to electrical property tests by the changes on diffusivity.

Reply:
Oh, I forgot to uncomment this. I was probing issues with test after when trying to figure out why the error message was no longer raised. Nice catch